java/java-cloud-run-hello-world/README.md

```diff
@@ -47,6 +47,23 @@ This sample demonstrates how to use the Cloud Code extension in IntelliJ.
 * [Install Cloud Code for IntelliJ](https://cloud.google.com/code/docs/intellij/install)
 * [Creating a new app](https://cloud.google.com/code/docs/intellij/create-run-app)
 
+## Testing
```

Comment:
Testing instructions should be moved to CONTRIBUTING.md similar to https://github.com/GoogleCloudPlatform/cloud-code-samples/blob/cloudrun-templates/golang/go-cloud-run-hello-world/CONTRIBUTING.md

Reply:
Ah, I should have pointed to https://github.com/GoogleCloudPlatform/cloud-code-samples/blob/cloudrun-templates/nodejs/CONTRIBUTING.md because nodejs is a bit ahead in using the `[lang]/.ci/` and `[lang]/CONTRIBUTING.md`.